django_project/certification/models.py

```diff
@@ -0,0 +1,302 @@
+    coursetype = models.ForeignKey(CourseType, on_delete=models.CASCADE)
+    trainingcenter = models.ForeignKey(TrainingCenter, on_delete=models.CASCADE)
+
+
+class CourseConvener(models.Model):
+    """ Register Course Convener """
+    name = models.CharField(
+        help_text="First name of course conveners",
+        max_length=150,
+        null=False,
+        blank=False
+    )
+
+    surename = models.CharField(
+        help_text="Surename course conveners",
+        max_length=150,
+        null=False,
+        blank=False
+    )
+
+    email = models.CharField(
+        help_text="Email address",
+        max_length=200,
+        null=False,
+        blank=False
+    )
+
+    slug = models.SlugField(unique=True)
+    objects = models.Manager()
+
+    # noinspection PyClassicStyleClass.
+    class Meta:
+        """ Meta class for Course convener """
+        app_label = 'certification'
+        ordering = ['name']
+
+    def save(self, *args, **kwargs):
+        """ Overloaded save method
+        :param args:
+        :param kwargs:
+        """
+        if not self.pk:
+            words = self.name.split()
+            filtered_words = [t for t in words if t.lower() not in STOP_WORDS]
+            # unidecode() represents special characters (unicode data) in ASCII
+            new_list = unidecode(' '.join(filtered_words))
+            self.slug = slugify(new_list)[:50]
+
+        super(CourseConvener, self).save(*args, **kwargs)
+
+    def __unicode__(self):
+        return u'%s' % self.name
+
+
+class CourseAttendee(models.Model):
+    """ Course Attendee model """
+    name = models.CharField(
+        help_text="First name course attendee",
+        max_length=200,
+        null=False,
+        blank=False
+    )
+
+    surename = models.CharField(
+        help_text="Surename course attendee",
+        max_length=200,
+        null=False,
+        blank=False
+    )
+
+    email = models.CharField(
+        help_text="Email address",
+        max_length=200,
+        null=False,
+        blank=False
+    )
+
+    slug = models.SlugField(unique=True)
+    objects = models.Manager()
+
+    # noinspection PyClassicStyleClass.
+    class Meta:
+        """ Meta class for Course attendee """
+        app_label = 'certification'
+        ordering = ['name']
+
+    def save(self, *args, **kwargs):
+        """ Overloaded save method
+        :param args:
+        :param kwargs:
+        """
+        if not self.pk:
+            words = self.name.split()
+            filtered_words = [t for t in words if t.lower() not in STOP_WORDS]
+            # unidecode() represents special characters (unicode data) in ASCII
+            new_list = unidecode(' '.join(filtered_words))
+            self.slug = slugify(new_list)[:50]
+
+        super(CourseAttendee, self).save(*args, **kwargs)
+
+    def __unicode__(self):
+        return u'%s' % self.name
+
+
+class CertifyingOrganisation(models.Model):
```

Comment:
![certificationschema](https://cloud.githubusercontent.com/assets/178003/23933490/8f01e046-0947-11e7-9f01-647a41ceab5e.png)

Updated diagram to reflect training center